pages.pt_BR/common/clang-format.md

```diff
@@ -0,0 +1,24 @@
+# clang-format
+
+> Formata automaticamente código C/C++/Java/JavaScript/Objective-C/Protobuf/C#.
+> Mais informações: <https://clang.llvm.org/docs/ClangFormat.html>.
+
+- Formata um arquivo e exibe o resultado para a `stdout` (saída padrão):
+
+`clang-format {{caminho/para/arquivo}}`
+
+- Formata um arquivo "in-place", ou seja, salvando nele mesmo:
```

Comment:
No changes on this part. Just wanted to say I really like these explanations for terms that usually are not translated. Quite useful! Congrats! :heart:

Reply:
Thanks. Sadly I'm not aware of a better translation for in-place.

Reply:
I believe your solution is the best one. To keep the term in English, as it is most used, and give it a short explanation.